cmd/textiled/main.go

```diff
@@ -68,6 +69,22 @@ var (
 			Key:      "addr.filecoin.api",
 			DefValue: "/ip4/127.0.0.1/tcp/5002",
 		},
+		"dnsDomain": {
+			Key:      "dns.domain",
+			DefValue: "",
+		},
+		"dnsZoneID": {
+			Key:      "dns.zone_id",
+			DefValue: "",
+		},
+		"dnsToken": {
+			Key:      "dns.token",
+			DefValue: "",
+		},
+		"defaultProjectHash": {
+			Key:      "project_hash",
+			DefValue: "QmSNWjbDkafwxWUGWTyP1ko4J1CW2WeNfeoiCuTUW2YnDY",
```

Comment:
What's the advantage of using a hash for project pages? It's gonna add a bunch of complexity. A project page will need a URL path off the gateway, which could just be mapped to the project subdomain.